tests/v2/validation/provisioning/rke1/README.md

```diff
@@ -0,0 +1,198 @@
+    "rootSize": "16",
+    "secretKey": "",
+    "securityGroup": ["open-all"],
+    "securityGroupReadonly": false,
+    "sessionToken": "",
+    "spotPrice": "0.50",
+    "sshKeyContents": "",
+    "sshUser": "ec2-user",
+    "subnetId": "subnet-ee8cac86",
+    "tags": "",
+    "type": "amazonec2Config",
+    "useEbsOptimizedInstance": false,
+    "usePrivateAddress": false,
+    "userdata": "",
+    "volumeType": "gp2",
+    "vpcId": "vpc-bfccf4d7",
+    "zone": "a"
+  }
+```
+
+### Azure
+```json
+  "azureNodeTemplate": {
+    "availabilitySet": "docker-machine",
+    "clientId": "",
+    "clientSecret": "",
+    "customData": "",
+    "diskSize": "30",
+    "dns": "",
+    "dockerPort": "2376",
+    "environment": "AzurePublicCloud",
+    "faultDomainCount": "3",
+    "image": "canonical:UbuntuServer:18.04-LTS:latest",
+    "location": "eastus2",
+    "managedDisks": false,
+    "noPublicIp": false,
+    "openPort": [
+      "6443/tcp",
+      "2379/tcp",
+      "2380/tcp",
+      "8472/udp",
+      "4789/udp",
+      "9796/tcp",
+      "10256/tcp",
+      "10250/tcp",
+      "10251/tcp",
+      "10252/tcp"
+    ],
+    "plan": "",
+    "privateIpAddress": "",
+    "resourceGroup": "",
+    "size": "Standard_D2_v2",
+    "sshUser": "azureuser",
+    "staticPublicIp": false,
+    "storageType": "Standard_LRS",
+    "subnet": "docker-machine",
+    "subnetPrefix": "192.168.0.0/16",
+    "subscriptionId": "",
+    "tenantId": "",
+    "type": "azureConfig",
+    "updateDomainCount": "5",
+    "vnet": "docker-machine-vnet"
+}
+```
+
+### Harvester
+```json
+"harvesterNodeTemplate": {
+    "cloudConfig": "",
+    "clusterId": "",
+    "clusterType": "",
+    "cpuCount": "2",
+    "diskBus": "virtio",
+    "diskSize": "40",
+    "imageName": "default/image-gchq8",
+    "keyPairName": "",
+    "kubeconfigContent": "",
+    "memorySize": "4",
+    "networkData": "",
+    "networkModel": "virtio",
+    "networkName": "",
+    "networkType": "dhcp",
+    "sshPassword": "",
+    "sshPort": "22",
+    "sshPrivateKeyPath": "",
+    "sshUser": "ubuntu",
+    "type": "harvesterConfig",
+    "userData": "",
+    "vmAffinity": "",
+    "vmNamespace": "default"
+}
+```
+
+### Linode
+```json
+"linodeNodeTemplate:" { 
+    "authorizedUsers": "",
+    "createPrivateIp": true,
+    "dockerPort": "2376",
+    "image": "linode/ubuntu20.04",
+    "instanceType": "g6-dedicated-8",
+    "label": "",
+    "region": "us-east",
+    "rootPass": "",
+    "sshPort": "22",
+    "sshUser": "root",
+    "stackscript": "",
+    "stackscriptData": "",
+    "swapSize": "512",
+    "tags": "",
+    "token": "",
+    "type": "linodeConfig",
+    "uaPrefix": "Rancher",
+}
+```
+
+## CustomCluster
+For custom clusters, the below config is needed, only AWS/EC2. It is important to note that you MUST use an AMI that already has Docker installed and the service is enabled on boot; otherwise, this will not work.
+**ensure you have nodeProviders in provisioningInput**
```

Comment:
Nit-picky, but can we change to `Ensure you have nodeProviders in provisioningInput.`